WriteHDLUtils.py

```diff
@@ -271,6 +271,27 @@ def getListsOfGroupedMemories(aProcModule):
     memList = list(aProcModule.upstreams + aProcModule.downstreams)
     portList = list(aProcModule.input_port_names + aProcModule.output_port_names)
 
+    #horrible hack to fix order of modules
+    ia=-1;
+    ib=-1;
+    for mem in memList:
+        if "_BE" in mem.var() :
+            ia=memList.index(mem);
+        if "_BF" in mem.var() :
+            ib=memList.index(mem);
+    if ia != -1 and ib != -1 and ib < ia :
+        memList[ia], memList[ib] = memList[ib], memList[ia]
+
+    ia=-1;
+    ib=-1;
+    for mem in memList:
+        if "L1PHIH_BC" in mem.var() :
+            ia=memList.index(mem);
+        if "L1PHIH_BD" in mem.var() :
+            ib=memList.index(mem);
+    if ia != -1 and ib != -1 and ib < ia :
+        memList[ia], memList[ib] = memList[ib], memList[ia]
```

Comment:
@aryd I'm assuming the order here comes from the order in the wiring files, so should the order ultimately be fixed in the TrackletConfigBuilder in CMSSW? I'm not proposing doing this immediately, but it seems like it would be better to eventually fix the issue where it originates, if possible.